addons/test_mail/tests/test_performance.py

```diff
@@ -1245,7 +1245,7 @@ def test_message_format_multi(self):
         """
         messages_all = self.messages_all.with_env(self.env)
 
-        with self.assertQueryCount(employee=26):  # test_mail: 25
+        with self.assertQueryCount(employee=27):  # test_mail: 25
```

Comment:
Why extra queries?

Reply:
From my understanding I think it comes from `starredPersonas` that uses `message_sudo.starred_partner_ids`.